pkg/git/Util.go

```diff
@@ -29,7 +29,7 @@ import (
 
 const (
 	GIT_BASE_DIR        = "/git-base/"
-	SSH_PRIVATE_KEY_DIR = "/ssh-keys/"
+	SSH_PRIVATE_KEY_DIR = "/git-base/ssh-keys/"
```

Comment:
extend it from `GIT_BASE_DIR` to make it explicit

Reply:
done